torchao/quantization/observer.py

```diff
@@ -52,6 +52,17 @@ class PerAxis(GranularityType):
     """
     axis: int
 
+@dataclass(frozen=True)
+class RowWise(GranularityType):
```

Comment:
n00b question, why is GranularityType a set of classes instead of an enum?

if it's for storing granularity settings later, maybe `GranularityType` isn't the right name?  That name screams enum to me.

Reply:
I think because some subclasses want to store more information, I.e.  PerAxis stores which axis and a PerBlock would store block_size in each dim

Reply:
call it `GranularityConfig` then?

Reply:
@jerryzh168 how does this sound, I am not sure if this API his heavily used else where or is it okay to just change the name?

Reply:
GranularityConfig sounds reasonable I think, it's OK to change the name, it's only used in observer right now

Reply:
or just `Granularity`

Reply:
Im going to do this in a follow up PR